currency/addbalance.cc.lua

```diff
@@ -0,0 +1,39 @@
+{{/*
+		Simple Currency System By GasInfinity
+        
+        Recommended Trigger: Regex: "^-(addbalance|addbal)"

```

Comment:
```suggestion
        Recommended Trigger: Regex: "\A-(addbal(ance)?)"
```
YAGPDB triggers are **multiline** by default. This means that `^` matches the start of a line rather than the start of a string, meaning `Hello\n-addbal` would match here. This also goes for `$`, but this isn't something we're interested in here. You should use `\A` instead to match the start of the input text.

Reply:
done